PhoneBook-App/UserInput.cs

```diff
@@ -1,4 +1,6 @@
 ﻿
+using System.Text.RegularExpressions;
+
 static class UserInput
 {
     static internal int GetInt(string Title)
```

Comment:
I'd lean towards always giving variables meaningful names. Make it as descriptive as possible: "input", "result", here I have to figure out what x means and even if it takes 4 seconds, it's 4 wasted seconds. 😊